src/rpc/misc.cpp

```diff
@@ -719,11 +719,6 @@ UniValue getaddressutxos(const UniValue& params, bool fHelp)
             + HelpExampleRpc("getaddressutxos", "{\"addresses\": [\"tmYXBYJj1K7vhejSec5osXK2QsGa5MTisUQ\"], \"chainInfo\": true}")
             );
 
-    if (!fExperimentalInsightExplorer) {
```

Comment:
Shouldn't this be left in, and instead have the conditional modified to:
```
if (!(fExperimentalInsightExplorer || fExperimentalLightWalletd)) {
```
like `getaddressmempool` above it?